packages/react-instantsearch-hooks-dom/src/widgets/__tests__/RefinementList.test.tsx

```diff
@@ -0,0 +1,3458 @@
+            >
+              <label
+                class="ais-RefinementList-label"
+              >
+                <input
+                  class="ais-RefinementList-checkbox"
+                  type="checkbox"
+                  value="Sony"
+                />
+                <span
+                  class="ais-RefinementList-labelText"
+                >
+                  Sony
+                </span>
+                <span
+                  class="ais-RefinementList-count"
+                >
+                  350
+                </span>
+              </label>
+            </li>
+            <li
+              class="ais-RefinementList-item"
+            >
+              <label
+                class="ais-RefinementList-label"
+              >
+                <input
+                  class="ais-RefinementList-checkbox"
+                  type="checkbox"
+                  value="Incipio"
+                />
+                <span
+                  class="ais-RefinementList-labelText"
+                >
+                  Incipio
+                </span>
+                <span
+                  class="ais-RefinementList-count"
+                >
+                  320
+                </span>
+              </label>
+            </li>
+            <li
+              class="ais-RefinementList-item"
+            >
+              <label
+                class="ais-RefinementList-label"
+              >
+                <input
+                  class="ais-RefinementList-checkbox"
+                  type="checkbox"
+                  value="KitchenAid"
+                />
+                <span
+                  class="ais-RefinementList-labelText"
+                >
+                  KitchenAid
+                </span>
+                <span
+                  class="ais-RefinementList-count"
+                >
+                  318
+                </span>
+              </label>
+            </li>
+            <li
+              class="ais-RefinementList-item"
+            >
+              <label
+                class="ais-RefinementList-label"
+              >
+                <input
+                  class="ais-RefinementList-checkbox"
+                  type="checkbox"
+                  value="Whirlpool"
+                />
+                <span
+                  class="ais-RefinementList-labelText"
+                >
+                  Whirlpool
+                </span>
+                <span
+                  class="ais-RefinementList-count"
+                >
+                  298
+                </span>
+              </label>
+            </li>
+          </ul>
+        </div>
+      </div>
+    `);
+
+    const firstCheckbox = container.querySelector(
+      '.ais-RefinementList-checkbox'
+    ) as HTMLInputElement;
+
+    expect(firstCheckbox).not.toBeChecked();
+
+    userEvent.click(firstCheckbox);
+
+    await wait(0);
+
+    expect(firstCheckbox).toBeChecked();
```

Comment:
Good idea. This assertion would actually fail because there has been one call on load already.

I've added two assertions and a comment to reflect it d3d41051.